src/mongoose_rabbit_worker.erl

```diff
@@ -168,9 +189,25 @@ make_chat_msg(From, To, UserMsg) ->
 is_user_online(online) -> true;
 is_user_online(offline) -> false.
 
--spec close_rabbit_connection(Connection :: pid(), Channel :: pid()) ->
+-spec establish_rabbit_connection(Opts :: #amqp_params_network{},
+                                  Host :: jid:server()) -> ok.
+establish_rabbit_connection(Opts, Host) ->
+    {ok, Connection} = amqp_connection:start(Opts),
+    {ok, Channel} = amqp_connection:open_channel(Connection),
+    mongoose_metrics:update(Host, ?RABBIT_CONNECTIONS_METRIC, 1),
```

Comment:
Moreover, all the call to `mongoose_metrics:update/2` could be extracted into a separate local function.

Reply:
I'm not sure that there is such a need. Isn't it readable enough?

Reply:
It's readable enough.